config/_default/privacy.yaml

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------
+# 
+# Privacy Settings
+#
+# See:  https://gohugo.io/about/hugo-and-gdpr/
+#
+# googleAnalytics:
+#  ananymizIP:  Enable anonymiation of IP addresses
+#  disable:  Set to true to disable googleAnalytics
+#  respectDoNotTrack:  Check for Do Not Track in headers
+#  useSessionStorage:  Store session information in storage and
+#                      not in cookies
+privacy:
```

Comment:
what and where is our "privacy policy" and does everyone know?
What do we want to say about our non-promise of privacy when using online? (For diagnostic purposes etc)

Reply:
This is new to us -- it is one of the areas I noticed in poking around.  I took an initial cut at settings, but I'm open to suggestions. I would like someone that understand GDPR to weigh in and suggest any needed changes.